src/dashboard/livetvtuner.js

```diff
@@ -74,16 +97,6 @@ define(["globalize", "loading", "libraryMenu", "dom", "emby-input", "emby-button
         })
     }
 
-    function getTabs() {
```

Comment:
why is this check gone?

Reply:
The only two tabs were the main page and the services page, which I removed.